docs/examples/teams/create-membership.md

```diff
@@ -9,7 +9,7 @@ client
     .setProject('5df5acd0d48c2') // Your project ID
 ;
 
-const promise = teams.createMembership('[TEAM_ID]', 'email@example.com', [], 'https://example.com');
+const promise = teams.createMembership('[TEAM_ID]', [], 'https://example.com');
```

Comment:
this example is misleading now. It does not contain an email in the example

Reply:
Yeah it doesn't make much sense, but it's because all of the email, userId and phone params are optional, and we don't show optional params in the examples